docker-compose.yml

```
@@ -2,88 +2,92 @@
 version: "3.7"
 
 secrets:
-  redis_conf:
-    file: ./secrets/redis.conf
   admiral_yml:
     file: ./secrets/admiral.yml
   mongo_root_passwd_txt:
     file: ./secrets/mongo-root-passwd.txt
+  redis_conf:
+    file: ./secrets/redis.conf
 
 x-admiral-template: &admiral-template
   build:
     context: .
     dockerfile: Dockerfile-admiral
-  image: admiral
-  init: true
+  deploy:
+    mode: replicated
+    replicas: 6
   environment:
     ADMIRAL_CONFIG_FILE: "/run/secrets/admiral.yml"
     ADMIRAL_CONFIG_SECTION: dev-mode
     ADMIRAL_WORKER_NAME: dev
+  image: admiral
+  init: true
   secrets:
     - source: admiral_yml
       target: admiral.yml
   volumes:  # map for development only, comment out otherwise
     - ./src/admiral:/usr/src/admiral/admiral
-  deploy:
-    mode: replicated
-    replicas: 6
 
 services:
-  redis:
-    image: "redis:alpine"
-    command:
-      - "redis-server"
-      - "/run/secrets/redis_conf"
+  celery-flower:
+    environment:
+      # TODO: https://github.com/cisagov/admiral/issues/19
+      # move into the secrets raft
+      CELERY_BROKER_URL: "redis://:fruitcake@redis:6379/0"
+      # TODO: https://github.com/cisagov/admiral/issues/18
+      # replace this outdated image
+    image: crgwbr/docker-celery-flower
```

Just fixing the alignment.

```suggestion
    # TODO: https://github.com/cisagov/admiral/issues/18
    # replace this outdated image
    image: crgwbr/docker-celery-flower
```